index.js

```diff
@@ -4,6 +4,11 @@ const fs = require("fs");
 const configYaml = require("config-yaml");
 const isDev = require("electron-is-dev");
 
+// Enable auto-reload in development if set
+try {
+  if (process.env.AUTO_RELOAD) require('electron-reloader')(module);
```

Comment:
I suggest to also add `isDev` if it's only in development.

```suggestion
  if (isDev && process.env.AUTO_RELOAD) require('electron-reloader')(module);
```